gdcdictionary/schemas/follow_up.yaml

```diff
@@ -2170,8 +2429,6 @@ properties:
           - "_terms_enum.yaml#/not_reported/common"
 
   risk_factor:
-    $ref:
```

Comment:
Here we are removing the terms ref but for comorbidity we are changing it from comorbidity to comorbidities. Is there a reason we are treating risk_factor differently? I would expect it to change similarly (i.e. risk_factor -> risk_factors).

Reply:
Good point, not technically necessary (since risk_factor is being deprecated), but added ref for consistency.